src/culture/culture.cpp

```diff
@@ -915,6 +915,21 @@ void update_research(sys::state& state, uint32_t current_year) {
 			}
 		}
 	}
+
+	if(state.cheat_data.instant_research && state.world.nation_get_current_research(state.local_player_nation)) {
```

Comment:
instant research doesn't need to be its own command because filling up the rp finishes it instantly anyways

Reply:
(and it comes at the cost of additional complexity here)

Reply:
Difference is that:
- `instant research` is active (it will continue to apply as long as its ON)
- whereas `rp` only does a one-time fillup

Reply:
yeah, but at the cost of this additional complexity here. Which is not worth it in my  opinion